fe/fe-core/src/main/java/org/apache/doris/nereids/rules/analysis/BindRelation.java

```diff
@@ -292,8 +292,8 @@ private LogicalPlan preAggForRandomDistribution(LogicalOlapScan olapScan) {
                 if (function == null) {
                     return olapScan;
                 }
-                Alias alias = new Alias(exprId, ImmutableList.of(function), col.getName(),
-                        olapScan.qualified(), true);
+                Alias alias = new Alias(StatementScopeIdGenerator.newExprId(), ImmutableList.of(function),
```

Comment:
add a new ctor for Alias accept four args as children, name, qualifier and nameFromChild